app/controllers/spina/admin/registers_controller.rb

```diff
@@ -15,6 +16,10 @@ def new
         @register = Spina::Register.new
       end
 
+      def edit
+        add_breadcrumb @register.name
```

Comment:
is it convention to use brackets for the args?

Reply:
Good spot!